src/AElf.Kernel.Types/Block/IBlock.cs

```diff
@@ -2,10 +2,9 @@
 {
     public interface IBlock : IBlockBase
     {
-        BlockHeader Header { get; set; }
-        BlockBody Body { get; set; }
-        long Height { get; set; }
+        BlockHeader Header { get; }
+        BlockBody Body { get; }
+        long Height { get; }
```

Comment:
remove height

Reply:
Remove and use the Header property to access height ? Some methods currently use IBlock.Height